spec/pivotal-ui-react/lists/lists_spec.js

```diff
@@ -31,7 +49,7 @@ describe('lists', function() {
   }
 
   describe('UnorderedList', function() {
-    itRenders(UnorderedList);
+    itRenders(UnorderedList, '#root ul');
```

Comment:
I think the `#root ul`was necessary for when the tests run in a visible browser window. It was breaking jasmine before.